conans/test/generators/cmake_test.py

```diff
@@ -172,6 +172,9 @@ def aux_cmake_test_setup_test(self):
     if(CONAN_EXPORTED)
         message(STATUS "Conan: called by CMake conan helper")
     endif()
+    if(CONAN_IN_LOCAL_CACHE)
+        message(STATUS "Conan: called inside local cache")
```

Comment:
Who is checking this message? If nobody is checking it, why adding it?

Reply:
It is added to the output here: https://github.com/conan-io/conan/pull/3450/files/45ee0fa7fb7d1e17c18ada5baa38778abdbb05c9#diff-4ace401e8414c1a6556986dc05a638d8, so the test need it too.

We can remove it from both places or leave it.